libs/books/feature/src/lib/book-search/book-search.component.ts

```diff
@@ -32,7 +32,7 @@ export class BookSearchComponent implements OnInit {
   }
 
   ngOnInit(): void {
-    this.store.select(getAllBooks).subscribe(books => {
+    this.bookSubscription = this.store.select(getAllBooks).subscribe(books => {
```

Comment:
Can you think of a better way to deal with this?

Reply:
Instead of creating an extra array to show the books in template, I have used async pipe now so that extra space is not needed and async pipe will take care of the unsubscribing also